app/src/main/java/com/example/plannr/admin/adminAdd/adminAddFragment.java

```diff
@@ -0,0 +1,93 @@
+package com.example.plannr.admin.adminAdd;
+
+import android.os.Bundle;
+
+import androidx.annotation.NonNull;
+import androidx.fragment.app.Fragment;
+
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
+import android.widget.CheckBox;
+import android.widget.EditText;
+import android.widget.TextView;
+
+import com.example.plannr.R;
+import com.example.plannr.databinding.FragmentAdminAddBinding;
+import com.example.plannr.databinding.FragmentFirstBinding;
+import com.example.plannr.databinding.FragmentLoginViewBinding;
+import com.example.plannr.services.DatabaseConnection;
+
+
+public class adminAddFragment extends Fragment {
+
+    private FragmentAdminAddBinding binding;
+    AdminAddPresenter presenter;
+    DatabaseConnection db;
+
+    public adminAddFragment() {
+        // Required empty public constructor
+        db = DatabaseConnection.getInstance();
```

Comment:
since this isn't related to the view, its related to data and stuff, it should actually live in the model/presenter (but it doesn't matter since this doesn't need to conform to MVP)

Reply:
wait lol you dont even use db in this file, you can totally remove this

Reply:
Ye i think i might have accidentally copied and pasted lol it ill remove it rn